app/controllers/coronavirus_form/are_you_off_work_ill_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::AreYouOffWorkIllController < ApplicationController
+  skip_before_action :check_first_question
```

Comment:
This skip check is only needed for the first question in the application

Reply:
Removed 👍